automation/tests/setup.sh

```diff
@@ -0,0 +1,315 @@
+#!/usr/bin/bash -x
+##!/usr/bin/expect  -d
+
+
+
+LOG_LOCATION=./logs
+exec > >(tee -i $LOG_LOCATION/install.log)
+exec 2>&1
+
+
+mkdir logs
+if [ ! -d ./logs ]
+then
+  mkdir ./logs
+fi
+
+if [ ! -d ./report ]
+then
+  mkdir ./report
+fi
+
+
+
+rm  -rf  $LOG_LOCATION/*
+rm  -rf ./report/result.txt
+
+
+
+helpFunction()
+{
+   echo ""
+   echo "Usage:  -h servername -u username -p passwd"
+   echo -e "\t-host name of the host server -u username of host -p passwd of user "
+   exit 1 # Exit script after printing help
+}
+
+MODULE="Attribute defaultMethods cacheconfig memcacheconf memory-cache-configue redis-cache-configuration in-Memory-cache-configuration native-persist-cacheconf Configuration-property fido2-configuration SMTP-configuration Logconfiguration LDAP-configuration couchbaseDB-configuration openID-connnect UMA OAuth-Scopes Statistics health server-Statistics scim-user-mgmt Organization-Configuration AuthServerHealth"
+
+reportgen () {
+	for i in $MODULE
+	do
+
+		if [[ `grep -ir "$i tests passed" $LOG_LOCATION/jans-cli_test.log` ]]
+		then
+			echo "$i Passed" >>./report/result.txt
+		else
+			echo "$i Failed" >>./report/result.txt
+
+		fi
+	done
+
+os_name=`uname -v | awk {'print$1'} | cut -f2 -d'-'`
```

Comment:
1. don't use html
2. Use markdown but don't write it here. Instead create a template markdown file in the directory. Call it and replace the temp values with the results. That should make it readable and easily updated. You would only be modifying the results instead of updating the whole doc in bash.

Reply:
this is done used  markdown template and updated result